GovUk.Frontend.Umbraco/ServiceCollectionExtensions.cs

```diff
@@ -19,8 +19,7 @@ public static class ServiceCollectionExtensions
     {
         public static IServiceCollection AddGovUkFrontendUmbraco(this IServiceCollection services)
         {
-            // Avoid adding scripts which require 'unsafe-inline' in the content security policy
-            return services.AddGovUkFrontendUmbraco(options => { options.AddImportsToHtml = false; });
+            return services.AddGovUkFrontendUmbraco(options => { });
```

Comment:
do we need to pass the options delegate parameter at all ?

Reply:
(here and elsewehre)

Reply:
Yes, otherwise it would call the same overload in an infinite loop.

It's worth retaining the overload that allows options to be passed in case a consuming project wants to do that.